content/en/blog/2021/migrate-alpha-security-policy/index.md

```diff
@@ -0,0 +1,273 @@
+
+1. Repeat the process for the next authentication policy.
+
+## `v1alpha1` authentication policy example
+
+Assume you have the following `v1alpha1` policy for the `httpbin` service in the `foo` namespace:
+
+{{< text yaml >}}
+apiVersion: authentication.istio.io/v1alpha1
+kind: Policy
+metadata:
+  name: example
+  namespace: "foo"
+spec:
+  targets:
+  - name: httpbin
+    ports:
+    - number: 8000
+  peers:
+  - mtls: {}
+  origins:
+  - jwt:
+      issuer: testing@example.com
+      jwksUri: "https://www.example.com/jwks.json"
+      triggerRules:
+      - includedPaths:
+        - prefix: "/admin/"
+        excludedPaths:
+        - exact: "/admin/status"
+  principalBinding: USE_ORIGIN
+{{< /text >}}
+
+Migrate the above authentication policy to `v1beta1` in the following ways:
+
+1. Assume the `httpbin` service in the `foo` namespace has the following workload selector:
+
+    {{< text yaml >}}
+    selector:
+      app: httpbin
+    ports:
+    - name: http
+      port: 8000
+      targetPort: 80
+    {{< /text >}}
+
+1. Create the `PeerAuthentication` to migrate the mTLS part:
+
+    {{< text yaml >}}
+    apiVersion: security.istio.io/v1beta1
+    kind: PeerAuthentication
+    metadata:
+      name: httpbin
+      namespace: foo
+    spec:
+      selector:
+        matchLabels:
+          app: httpbin
+    mtls:
+      # Use PERMISSIVE by default for maximum backward compatibility
+      mode: PERMISSIVE
+    portLevelMtls:
+      # This should be the workload port 80, not the service port 8000
+      80:
+        mode: STRICT
+    {{< /text >}}
+
+1. Create the `RequestAuthentication` and `AuthorizationPolicy` to migrate the JWT part:
+
+    {{< text yaml >}}
+    apiVersion: security.istio.io/v1beta1
+    kind: RequestAuthentication
+    metadata:
+      name: example-httpbin
+      namespace: foo
+    spec:
+      selector:
+        matchLabels:
+          app: httpbin
+      jwtRules:
+      - issuer: testing@example.com
+        jwksUri: "https://www.example.com/jwks.json"
+    ---
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: example-httpbin
+      namespace: foo
+    spec:
+      # Use DENY action to explicitly deny requests without JWT token
+      action: DENY
+      selector:
+        matchLabels:
+          app: httpbin
+      rules:
+      - from:
+        - source:
+            # This makes sure requests without JWT token will be denied
+            notRequestPrincipals: ["*"]
+        to:
+        - operation:
+            # This should be the workload port 80, not the service port 8000
+            ports: ["80"]
+            # The path is converted from the trigger rule
+            notPaths: ["/admin/status"]
+            paths: ["/admin/*"]
+    {{< /text >}}
+
+1. Apply the `PeerAuthentication`, `RequestAuthentication` and `AuthorizationPolicy` and monitor the traffic to make
```

Comment:
We should also mention "1. Remove the old policies " for cleanup after this, right?

Reply:
Same for the authz part.